packages/webapp/pages/devcard.tsx

```diff
@@ -268,7 +268,7 @@ const Step2 = ({
             <Button
               className="mt-4 btn-secondary"
               buttonSize="small"
-              onClick={() => copyEmbed}
+              onClick={() => copyEmbed()}
```

Comment:
Why not just:
```ts
onClick={copyEmbed}
```

Reply:
Types don't match and didn't want to break the other implementations of the useCopyLink hook.
We should probably add it for a future fix though 👀